dashboard/app/views/levels/_external_link.html.haml

```diff
@@ -0,0 +1,7 @@
+.container
+  - if @level.markdown_instructions
+    %div= render(inline: @level.markdown_instructions, type: :md)
+
+  %h4= link_to @level.link_title, @level.url, target: '_blank'
+
+  %div{style: 'padding-top: 100px'}= render(inline: t('external_links.disclaimer_md'), type: :md)
```

Comment:
This is really clever, but doesn't actually prevent the translator from inserting markup ([Redcarpet](https://github.com/vmg/redcarpet) allows html tags).

Since we already have i18n strings with markup, let's follow that pattern for now.  @tanyaparker has a Pivotal item to sanitize HTML in i18n strings: https://www.pivotaltracker.com/story/show/119412087.